coops-ui/public/config/configuration.json

```diff
@@ -1,6 +1,6 @@
 {
-  "API_URL": "https://legal-api-dev.pathfinder.gov.bc.ca/api/v1/businesses/",
-  "AUTH_URL": "https://coops-dev.pathfinder.gov.bc.ca/auth/",
+  "API_URL": "http://localhost:2015api/v1/businesses/",
```

Comment:
Please don't check in these changes. There is a second config file for using `localhost:2015`.